.github/workflows/test-suite.yml

```
@@ -130,18 +130,21 @@ jobs:
-          ANTHROPIC_API_KEY: dummyapikey
-          GOOGLE_GENERATIVE_AI_API_KEY: dummyapikey
-          COHERE_API_KEY: dummyapikey
-          PINECONE_API_KEY: dummyapikey
-          ASTRA_DB_ENDPOINT: https://dummy.local
-          ASTRA_DB_TOKEN: dummyapikey
-          CLOUDFLARE_API_TOKEN: dummyapikey
-          CLOUDFLARE_ACCOUNT_ID: dummyapikey
-          KV_REST_API_URL: https://dummy.local
-          KV_REST_API_TOKEN: dummyapikey
+          # LLM API Keys
+          OPENAI_API_KEY: ${{ secrets.OPENAI_API_KEY }}
```

actionlint (1.7.12)

[error] property "openai_api_key" is not defined in object type {actions_runner_debug: string; actions_step_debug: string; github_token: string; turbo_team: string; turbo_token: string}

(expression)